go.mod.sri

```diff
@@ -1 +1 @@
-sha256-L7V4Wef0rDY+M3XyOuRpma03Eq2ONZolKB3obU9M8Ic=
```

Comment:
Hmm, why did the go.mod SRI hash change? It should only depend on the contents of go.mod, not flake.nix...

Ah, it's because a bot sent me https://github.com/tailscale/tailscale/pull/9982/files for a go.mod change, and I didn't see it until now! Okay that's fine then, we can merge this and I'll fix up the bot change afterwards.

Reply:
Oops, sorry, I didn't realize the hash existed in multiple places!